diff --git a/pkg/blobserver/google/cloudstorage/storage.go b/pkg/blobserver/google/cloudstorage/storage.go
index 93d6ced..3be346e 100644
--- a/pkg/blobserver/google/cloudstorage/storage.go
+++ b/pkg/blobserver/google/cloudstorage/storage.go
@@ -128,7 +128,7 @@ func newFromConfig(_ blobserver.Loader, config jsonconfig.Obj) (blobserver.Stora
 	}

 	if cacheSize != 0 {
-		gs.cache = memory.NewCache(cacheSize)
+		gs.cache = memory.NewCache(5<<20)
 	}

 	bi, err := gs.client.BucketInfo(bucket)
